var/spack/repos/builtin/packages/py-or-tools/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyOrTools(CMakePackage):
+    """This project hosts operations research tools developed at
+    Google and made available as open source under the Apache 2.0
+    License."""
+
+    homepage = "https://developers.google.com/optimization/"
+    url      = "https://github.com/google/or-tools/archive/v7.8.tar.gz"
+
+    version('7.8', sha256='d93a9502b18af51902abd130ff5f23768fcf47e266e6d1f34b3586387aa2de68')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-numpy@1.0:', type=('build', 'run'))
+    depends_on('py-packaging@16.0:', type=('build', 'run'))
+    depends_on('py-protobuf@3.12.2')
+    depends_on('protobuf@3.12.2')
+    depends_on('py-six')
+    depends_on('gflags@2.2.2')
+    depends_on('glog@0.4.0')
+    depends_on('protobuf@3.12.2')
+    depends_on('abseil-cpp@20200225.2')
+    depends_on('cbc@2.10.5')
+    depends_on('cgl@0.60.3')
+    depends_on('clp@1.17.4')
+    depends_on('osi@0.108.6')
+    depends_on('coinutils@2.11.4')
```

Comment:
All of these specific versions are required? Or can newer versions also work?

Reply:
Possible that newer versions are okay, but I'm working off what they've placed in Dependencies.txt for now.